test/integration/integration_test.cc

```diff
@@ -157,6 +157,55 @@ TEST_P(IntegrationTest, PerWorkerStatsAndBalancing) {
   check_listener_stats(0, 1);
 }
 
+// Make sure we have correctly specified per-worker performance stats.
```

Comment:
I think this comment is wrong?

Reply:
yeah it is wrong, updated to the correct version on 5034e6e7bec0bb960b0a89e0ada66b94ebf62205